ext/DashBasePlotsExt.jl

```diff
@@ -0,0 +1,9 @@
+module DashBasePlotsExt
+
+import Plots
+import DashBase
+
+
+DashBase.to_dash(p::Plots.Plot{Plots.PlotlyBackend}) = (data = Plots.plotly_series(p), layout = Plots.plotly_layout(p))
```

Comment:
Does `Plots` have support for Plotly animation `frames` settings?

https://plotly.com/python/animations/

Reply:
I think currently not, but we could also pass a `PlotlyBase.SyncPlot` here.

Reply:
I'm ok leaving animation frames support from `Plots` unresolved for now.